### DISCUSSION TITLE
filter by Date Range

### DISCUSSION THREAD

**Discussion by NeloReis** [Budibase/budibase#2859] [Help]
Hi I'm trying to create a date range to filter data from a data provider.
the provided date range  element is limited to a few options ( and even that one, i don't know how to make it work)
are there other options? to create a Date Range filter? or just the date RAnge element. and if so .. how do I use that thing?
Thanks

**Comment by mjashanks** [comment]
Hey @NeloReis 

To add a Date Range filter, use 2 date pickers.
1. Surround your DataProvider in a `Form`
2. Above your dataprovider add a Date Picker - set the "Field" setting to `fromDate`
3. Repeat for a `toDate` field
4. Use these fields in the `Define Filters` for your data provider.

Video below should show my setup.

https://user-images.githubusercontent.com/3524181/135885291-5584aa92-3d57-4d58-aa98-1db35bf6309c.mp4
